packages/cli/src/util/telemetry/index.ts

```diff
@@ -157,8 +157,16 @@ export class TelemetryClient {
     return;
   }
 
-  trackFlagHelp() {
-    this.trackCliFlag('help');
+  trackCliFlagHelp(command: string, subcommands?: string | string[]) {
```

Comment:
The card describes making this `protected`, but for that to work we'd need to add

```
trackCliFlagHelp(command: string, subcommands?: string | string[]) {
  super.trackCliFlagHelp(command, subcomands)
}
```

to every child class. This seems like overkill to protect from this method being called on instances of the base class so I called an audible and made the method public. Happy to do otherwise if there's a compelling reason.